app/Http/Controllers/Multiplayer/Rooms/Playlist/ScoresController.php

```diff
@@ -64,7 +64,7 @@ public function index($roomId, $playlistId)
             ->limit($limit)
             ->getWithHasMore();
 
-        $transformer = ScoreTransformer::newSolo();
+        $transformer = new ScoreTransformer(false);
```

Comment:
this probably doesn't need explicit specification if we just add multiplayer score link support for legacy format 🤔